### ISSUE TITLE
mysql root password preseed doesn't work

### ISSUE THREAD

**Issue by abizer** [ocf/puppet#471]
the preseed file we include for debconf (https://github.com/ocf/puppet/blob/master/modules/ocf_mysql/manifests/init.pp#L6) has two problems. First, we're now on mariadb-server-10.1, so the package name for debconf needs to be updated, but more importantly, the mariadb-server-10.1 package templates doesn't appear to support mysql-server/root_password and root_password_again debconf options at all:

```bash
root@e5a1d73f257e:/# sudo debconf-show mariadb-server-10.1                                                                                                                               
  mariadb-server-10.1/nis_warning:
  mariadb-server-10.1/old_data_directory_saved:
  mariadb-server-10.1/postrm_remove_databases: false
```

unless I'm interpreting this incorrectly, we'll need to change the way we preseed the root password and do initial grants.

**Comment by jvperrin** [comment]
From [this Debian bug comment](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=865931#28) it looks like we shouldn't need to preseed it any more since the local root user can log in over a unix socket. We also have the root password set up in the database already so when we migrate to a new host then when we import from backups (or from the current database host), then the root password should already be set, so I don't think we need to preseed or do any fancy puppet config for it.

Thoughts? I'm going to close this for now and delete the preseed file from the private share (we can remake it knowing the root password), I think we can just not do it, but I'm open to feedback, feel free to reopen this.

**Comment by abizer** [comment]
Yeah I don't think this is necessary anymore. Can be deleted and closed.
